containers/docker/docker-compose-postgres.yml

```diff
@@ -15,8 +15,13 @@ services:
     image: docker.io/pglombardo/pwpush:latest
     environment:
       DATABASE_URL: 'postgres://pwpush_user:pwpush_passwd@db:5432/pwpush_db'
+      # THRUSTER_TLS_DOMAIN: sub_domain.example.com # Uncomment to enable HTTPS
```

Comment:
We can define a new key in `config/settings.yml` to manage easier.

Reply:
I have removed `THRUSTER` from `THRUSTER_TLS_DOMAIN`.

I have noticed that `PWP__HOST_DOMAIN` is already used, so we can define `TLS_DOMAIN`  by using it. As a result, [I have updated to use `PWP__HOST_DOMAIN`.](https://github.com/pglombardo/PasswordPusher/pull/3501/files#diff-5c33012ced24ab9ec9f1c912691a6d2f6d69e1a76c2f77c3419ca96cdd6dc55fR35-R37)